src/Engine/Data/Mesh.inl

```diff
@@ -456,6 +456,74 @@ void IndexedGeometry<T>::render( const ShaderProgram* prog ) {
     }
 }
 
+////////////////  MultiIndexedGeometry  ///////////////////////////////
+
+template <typename T>
+MultiIndexedGeometry<T>::MultiIndexedGeometry( const std::string& name,
+                                               typename base::CoreGeometry&& geom,
+                                               typename base::MeshRenderMode renderMode ) :
+    base( name, renderMode ) {
+    loadGeometry( std::move( geom ) );
+}
+
+template <typename T>
+void MultiIndexedGeometry<T>::loadGeometry( T&& mesh ) {
+    m_indices.clear();
+
+    base::loadGeometry_common( std::move( mesh ) );
+    /// \todo HERE
+    // indices
+    //  base::m_mesh.attach( IndicesObserver( this ) );
+}
+
+template <typename T>
+void MultiIndexedGeometry<T>::updateGL_specific_impl() {
+    //    if ( !m_indices )
+    //    {
+    //        m_indices      = globjects::Buffer::create();
+    //        m_indicesDirty = true;
+    //    }
+    //    if ( m_indicesDirty )
+    //    {
+    //        /// this one do not work since m_indices is not a std::vector
+    //        // m_indices->setData( m_mesh.m_indices, GL_DYNAMIC_DRAW );
+    //        m_numElements =
+    //            base::m_mesh.getIndices().size() *
+    //            base::CoreGeometry::IndexType::RowsAtCompileTime;
+    //
+    //        m_indices->setData(
+    //            static_cast<gl::GLsizeiptr>( base::m_mesh.getIndices().size() *
+    //                                         sizeof( typename base::CoreGeometry::IndexType ) ),
+    //            base::m_mesh.getIndices().data(),
+    //            GL_STATIC_DRAW );
+    //        m_indicesDirty = false;
+    //    }
+    //    if ( !base::m_vao ) { base::m_vao = globjects::VertexArray::create(); }
+    //    base::m_vao->bind();
+    //    base::m_vao->bindElementBuffer( m_indices.get() );
+    //    base::m_vao->unbind();
+    /// \todo implement !
```

Comment:
As this prevent the use of `MultiIndexedGeometry`, we should discuss and implement it before accepting this PR.

Reply:
Why is this preventing the use of `MultiIndexGeometry` ? It is used in the PR and all applications work fine. Can you be more specific ?

Reply:
well there is no MultiIndexedGeometry on the Engine side, they are drawn as Triangle or Polymeshes.